lives-of-primitive-quakers/modernized/05-chapter-4.adoc

```diff
@@ -359,7 +367,7 @@ and that we now suffer for your holy name, and for your honour and justice,
 and for your truth and holiness!
 O Lord, you know we are resolved to perish, rather than to lose one grain hereof!
 Amen, amen!`"^
-footnote:[Sewel's History, (Lindfield edition,) vol.ii.pp. 420, 425.]
+footnote:[[.book-title]#Sewel's History#, (Lindfield edition,) vol.ii.pp. 420, 425.]
```

Comment:
while you're changing things, let's add some spaces and make it:

```
vol. ii. pp. 420
```

Seems better, right?